File: src/main/java/net/atos/zac/app/zaken/model/RESTCoordinates.java
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.app.zaken.model;

public class RESTCoordinates {

  public RESTCoordinates() {}

  public RESTCoordinates(final double x, final double y) {
    this.x = x;
    this.y = y;
  }

  public double x;

  public double y;
}

File: src/main/java/net/atos/zac/app/zaken/model/RESTCoordinates.kt
/*
 * SPDX-FileCopyrightText: 2021 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.app.zaken.model

class RESTCoordinates {
    constructor()

    constructor(x: Double, y: Double) {
        this.x = x
        this.y = y
    }

    var x: Double = 0.0

    var y: Double = 0.0
}
